### PR TITLE
[RLlib] minor cleanup of connector to/from state APIs

### PR DESCRIPTION
**Pull request by gjoliver**
## Why are these changes needed?

Also better error messages.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [*] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/connectors/connector.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def in_eval(self):
     def __str__(self, indentation: int = 0):
         return " " * indentation + self.__class__.__name__
 
-    def to_state(self) -> Tuple[str, List[Any]]:
+    def to_state(self) -> Tuple[str, Any]:
```
Comment (kouroshHakha) [inline rllib/connectors/connector.py:109]: Can you add a doc on what str and Any should be?

File: rllib/utils/policy.py [hunk 2]
```diff
@@ -146,7 +146,7 @@ def load_policies_from_checkpoint(
         policy_class = get_tf_eager_cls_if_necessary(
             policy_spec.policy_class, merged_config
         )
-
+        print(merged_config["_disable_execution_plan_api"])
```
Comment (kouroshHakha) [inline rllib/utils/policy.py:149]: remove the print?

**Review by kouroshHakha — APPROVED** [review]

File: rllib/connectors/connector.py (hunk 1 above)
Comment (gjoliver) [reply rllib/connectors/connector.py:109]: done

File: rllib/utils/policy.py (hunk 2 above)
Comment (gjoliver) [reply rllib/utils/policy.py]: oh shoot, thanks for catching this.

**Comment by gjoliver** [comment]
tests all pass. merging.

**Pull request merged**